bin/util.py

```diff
@@ -835,9 +845,22 @@ def _try_wait(self, wait_flags):
                 self.rusage = None
             return (pid, sts)
 
+def default_exec_code_map(code):
+    if code == 0:
+        return ExecCode.ACCEPTED
+    if code == -9:
+        return ExecCode.TIMEOUT
+    return ExecCode.ERROR
+
+def icpc_exec_code_map(code):
+    if code == config.RTV_AC:
+        return ExecCode.ACCEPTED
+    if code == config.RTV_WA:
+        return ExecCode.REJECTED
+    return ExecCode.ERROR
```

Comment:
Hmmm, not sure whether we currently handle timeout for validators and/or whether we should.
Probably it's useful for users to know the error was a validation timeout rather than a generic 'error'?

Reply:
it is probably useful, but the main reason to add this was because generators etc. handled this and i thought it would fit better here